src/scaffold/store.ts

```diff
@@ -95,37 +99,32 @@ export const useStore = create<Store>((set) => {
       updateTemplateOptions() {
         dispatch(set, (state) => ({
           ...state,
-          templateOptions: ['spinner'],
+          isLoading: true,
         }));
         const packageName = useStore.getState().state.options.template;
         vscode.postMessage({
           command: 'getTemplatesOptions',
           packageName,
         });
+        dispatch(set, (state) => ({
```

Comment:
Why?

We already `dispatch` 4 lines above. This is redundant. Only one of them should be kept.